tests/test_solph_network_classes.py

```diff
@@ -92,14 +92,55 @@ def test_flow_with_fix_and_min_max():
         solph.flows.Flow(fix=[1, 3], max=[0, 5], min=[4, 9])
 
 
+def test_infinite_values():
+    msg1 = "nominal_value must be a finite value"
+    msg2 = "max must be a finite value"
+    with pytest.raises(AssertionError, match=msg1):
+        solph.flows.Flow(nominal_value=float("+inf"))
+    with pytest.raises(AssertionError, match=msg2):
+        solph.flows.Flow(max=float("+inf"))
+
+
+def test_attributes_needing_nominal_value_get_it():
+    with pytest.warns(
+        SuspiciousUsageWarning, match="If fix is set in a flow "
+    ):
+        solph.flows.Flow(fix=0.3)
+
+    with pytest.warns(
+        SuspiciousUsageWarning, match="If max is set in a flow "
+    ):
+        solph.flows.Flow(max=0.3)
+
+    with pytest.warns(
+        SuspiciousUsageWarning, match="If min is set in a flow "
+    ):
+        solph.flows.Flow(min=0.3)
+
+    with pytest.warns(
+        SuspiciousUsageWarning, match="If summed_max is set in a flow "
+    ):
+        solph.flows.Flow(summed_max=0.3)
+
+    with pytest.warns(
+        SuspiciousUsageWarning, match="If summed_min is set in a flow "
+    ):
+        solph.flows.Flow(summed_min=0.3)
+
+
 def test_min_max_values_for_bidirectional_flow():
-    a = solph.flows.Flow(bidirectional=True)  # use default values
-    b = solph.flows.Flow(bidirectional=True, min=-0.9, max=0.9)
+    a = solph.flows.Flow(
+        bidirectional=True,
+        nominal_value=1,
```

Comment:
This nominal_value in this Flow is `None` and not `1`, so you do not have to set it, because `None` is the default value.